src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        action="#"
+        method="get"
+        class="big__search__form"
+        data-qa="big"
+      >
+        <input
+          name="bigForm"
+          type="search"
+          class="big__search search"
```

Comment:
Add default style for search class and change this style for small__search class (you don't need big__search class)